route/build/src/ncio_utils.f90

```diff
@@ -14,6 +14,7 @@ MODULE ncio_utils
 public::check_attr
 public::check_variable
 public::put_global_attr
+public::get_variable_rank
```

Comment:
When I see rank in a method name I figure it has something to do with MPI. But this is really about getting the PIO/netCDF ID. So I would recommend changing the name a bit.

Reply:
It looks like both ndims and rank is used. So you could change to get_variable_ndims or add a comment to the subroutine.

Reply:
changed "rank" to "ndims".